### PR TITLE
Android pages

### PR DESCRIPTION
**Pull request by BAkosM**
**Do not DELETE this branch just merge it!**

### CODE REVIEW

File: GreenRace_Mobil/app/src/main/java/com/example/greenrace/UserRegisterDto.kt [hunk 1]
```diff
@@ -3,6 +3,7 @@ package com.example.greenrace
 import com.fasterxml.jackson.annotation.JsonCreator
 import com.fasterxml.jackson.annotation.JsonProperty
 import com.google.gson.annotations.SerializedName
+import java.util.Date
```
Comment (lacgondi) [inline GreenRace_Mobil/app/src/main/java/com/example/greenrace/UserRegisterDto.kt]: use the kotlin version if possible

File: GreenRace_Mobil/app/src/main/java/com/example/greenrace/MainActivity.kt [hunk 2]
```diff
@@ -45,10 +45,10 @@ class MainActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
 
-        val intent = Intent(this@MainActivity, MapsActivity::class.java)
-        startActivity(intent)
+        //val intent = Intent(this@MainActivity, MapsActivity::class.java)
```
Comment (lacgondi) [inline GreenRace_Mobil/app/src/main/java/com/example/greenrace/MainActivity.kt]: Remove unused code

**Review by lacgondi — CHANGES_REQUESTED** [review]

**Comment by BAkosM** [comment]
Fixed

**Review by lacgondi — APPROVED** [review]
nice job

**Pull request merged**